shared/rocket/build.gradle.kts

```diff
@@ -1,19 +1,54 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+
 plugins {
     alias(libs.plugins.quanti.kmp.library)
     alias(libs.plugins.quanti.android.detekt)
     alias(libs.plugins.quanti.android.ktlint)
     alias(libs.plugins.kover)
+    alias(libs.plugins.kotlin.multiplatform)
 }
 
 android {
     namespace = "cz.quanti.rocketrepository"
 }
 
+val libName = "Shared"
+val iosLibFolder = "XCFrameworks/lib"
+
 multiplatformSwiftPackage {
```

Comment:
so this is old way, not supporting simulator?

Reply:
No the old way is integrated into the project as a Package directly through dependency. Thus the KMP module is standalone library on git

Reply:
try without this